lib/rbnacl/hash/blake2b.rb

```diff
@@ -170,14 +170,9 @@ def digest
       end
 
       # The crypto_generichash_blake2b_state struct representation
-      # ref: jedisct1/libsodium/src/libsodium/include/sodium/crypto_generichash_blake2b.h#L23
+      # ref: jedisct1/libsodium/blob/c8d15621c75f49cf13463e83d37df74c7b5969f4/src/libsodium/include/sodium/crypto_generichash_blake2b.h#L23-L25
```

Comment:
Rubocop is complaining about the line length. You can either shut it off for this comment block:

```
# rubocop:disable
# The crypto_generichash_blake2b_state struct representation
# ref: jedisct1/libsodium/src/libsodium/include/sodium/crypto_generichash_blake2b.h#L23	      # ref: jedisct1/libsodium/blob/c8d15621c75f49cf13463e83d37df74c7b5969f4/src/libsodium/include/sodium/crypto_generichash_blake2b.h#L23-L25
# rubocop:enable
```

...or truncate the hash perhaps?